FBSimulatorControl/Strategies/FBSimulatorBootStrategy.m

```diff
@@ -421,8 +421,9 @@ - (FBProcessInfo *)launchdSimPresentWithAllRequiredServices:(NSError **)error
 - (NSArray<NSString *> *)requiredLaunchdServicesToVerifyBooted
 {
   FBControlCoreProductFamily family = self.simulator.productFamily;
+  id<FBControlCoreGlobalConfiguration_OS> version = self.simulator.osConfiguration;
   if (family == FBControlCoreProductFamilyiPhone || family == FBControlCoreProductFamilyiPad) {
-    if (FBControlCoreGlobalConfiguration.isXcode8OrGreater) {
+    if (FBControlCoreGlobalConfiguration.isXcode8OrGreater && [[version versionNumber] doubleValue] >= 9.0) {
```

Comment:
The check itself looks fine, but you can avoid doing this via a `double` and instead use `NSDecimalNumber`, avoiding any hairyness with floating-point numbers.

```[version.versionNumber isGreaterThanOrEqualTo:[NSDecimalNumber numberWithString:@"9.0"]]```